mk/genlink-config.mk

```diff
@@ -18,17 +18,61 @@
 ##
 
 ifeq ($(DEVICE),)
-$(error no DEVICE specified for linker script generator)
+$(warning no DEVICE specified for linker script generator)
```

Comment:
Why is this now a warning instead of an error ? Wouldn't the following awk commands fail if DEVICE isn't specified ?